mrmustard/physics/ansatz/polyexp_ansatz.py

```diff
@@ -208,6 +209,56 @@ def num_derived_vars(self) -> int:
     def num_vars(self):
         return self.A.shape[-1]
 
+    @property
+    def PS(self) -> PolyExpAnsatz:
+        r"""
+        The ansatz acting on real (i.e., phase-space) variables.
+        """
+        n = self.A.shape[-1]
+        if n % 2:
+            raise ValueError(
+                f"A phase space ansatz must have even number of indices. (n={n} is odd)"
+            )
+
+        if self.c.shape == ():
+            In = math.eye(n // 2, dtype=math.complex128)
+            W = math.block([[In, -1j * In], [In, 1j * In]]) / complex(
+                math.sqrt(2, dtype=math.complex128) * settings.HBAR
+            )
```

Comment:
```suggestion
            W = math.conj(math.rotmat(n // 2)) / settings.HBAR
```
this should be equivalent